docs/source/guides/faq.md

```diff
@@ -0,0 +1,121 @@
+---
+title: Frequently Asked Questions
+description: Common questions asked at each stage of GraphQL adoption
+---
+
+Everyone has questions about how to properly set up a GraphQL schema, but not all questions are alike. In different stages of development, different things matter. This guide will questions that people commonly have have at every step along the journey to GraphQL in production.
+
+## Learning GraphQL
+
+> People in this phase are just beginning to learn GraphQL. They are trying to get a sample project up and running.
+
+#### Why use GraphQL?
+
+GraphQL can make a difference in nearly every area of development: from improving developer experience with quality tooling to improving client performance by reducing bundle sizes. Read more about the benefits of GraphQL [here](../fundamentals/benefits.html).
+
+#### Where can I learn GraphQL?
+
+There are a number of resources available to learn GraphQL. If you're looking to get started learning the basics, check out [GraphQL.org](https://graphql.org). To get started with `apollo-server`, a production-ready graphql server, check out the [getting started](https://www.apollographql.com/docs/apollo-server/getting-started.html) guide. For `react-apollo`, the React integration for apollo-client, read [this guide](https://www.apollographql.com/docs/react/essentials/get-started.html). For more advanced topics, the guides on this site and the [Apollo blog](https://blog.apollographql.com) are great places to learn and keep up with the latest developments.
+
+#### How can I host my schema online?
+
+A great tool for learning and building small projects is [Glitch](https://glitch.com). Glitch allows development of a schema in the browser, and even supports cloning from and pushing to GitHub. Glitch provides a public endpoint that projects can query against. To get started with building a GraphQL schema, try using and remixing the [Apollo Launchpad](https://glitch.com/~apollo-launchpad) project.
+
+GraphQL schemas written with `apollo-server` can be deployed anywhere that other Node.js projects can be deployed. `apollo-server` even has variants to support serverless deployment with AWS Lambda.
+
+There are deployment guides currently written for [Heroku](https://www.apollographql.com/docs/apollo-server/deployment/heroku.html), [Lambda](https://www.apollographql.com/docs/apollo-server/deployment/lambda.html), and [Now](https://www.apollographql.com/docs/apollo-server/deployment/now.html)
+
+#### How do I connect my client app to my schema?
+
+There are many tools available to connect any kind of client to a schema. The most common are [Apollo Client](https://www.apollographql.com/docs/react/) for JavaScript clients,
```

Comment:
Instead of being diplomatic ("there are many tools available...the most common are"), we should be authoritative about recommending Apollo.